kernel/src/actions/deletion_vector.rs

```diff
@@ -402,6 +414,26 @@ mod tests {
         assert_eq!(bools, expected);
     }
 
+    #[test]
+    fn test_sv_to_bools() {
```

Comment:
I found that it ran just fine on my machine, whereas the other one took a minute.

Giving an algorithmic complexity is hard to say tbh.  I go into it in this [comment](https://github.com/delta-io/delta-kernel-rs/pull/568#discussion_r1874613741), but the TLDR is that the zero-initialized large sv vector is fast to allocate. Just get a bunch of pages from the OS. The other one is slow because rust has to actually read the pages and set it to true.

I can disable it for consistency.

Reply:
ohhh right i recall. how about we leave it enabled then and just leave that comment here in the code with it?

Reply:
The difference is pretty staggering lol.  0.85s vs 40s on my machine!!!

Reply:
yea let's not keep the slow one. kinda nice we got some coverage through the other path now tho :)